data/libcaja-user-share.caja-extension.in.in

```diff
@@ -1,7 +1,7 @@
 [Caja Extension]
 Icon=user-share
-_Name=User Share
-_Description=Integrates with user-share
+Name=User Share
+Description=Integrates with user-share
 Copyright=Copyright (C) 2005 William Jon McCann <mccann@jhu.edu>
```

Comment:
Looks like that this translations are missing.
```
diff -uprN mate-user-share-orig.pot mate-user-share.pot > diff-pot-po-dir.diff

<cut>
-#: ../data/libcaja-user-share.caja-extension.in.in.h:1
-msgid "User Share"
+#: data/mate-user-share-properties.desktop.in:3 src/caja-share-bar.c:99
+msgid "Personal File Sharing"
 msgstr ""
 
-#: ../data/libcaja-user-share.caja-extension.in.in.h:2
-msgid "Integrates with user-share"
+#: data/mate-user-share-properties.desktop.in:4
+msgid "Preferences for sharing of files"
 msgstr ""

<cut>
```
All other translations are there.